flink-connectors/flink-connector-elasticsearch7/pom.xml

```diff
@@ -51,6 +51,12 @@ under the License.
 			<scope>provided</scope>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-connector-base</artifactId>
+			<version>${project.version}</version>
+		</dependency>
```

Comment:
Why did you add this dependency?

Reply:
I initially forgot why I added this, but now I recall :)

The problem is that elasticsearch connector requires flink-connector-base for `DeliveryGuarantee` class. Before this pr, the connector is depending on this package transitively through flink-table-api-java-bridge -> flink-table-api-java -> flink-table-common -> flink-connector-files -> flink-connector-base. Now that there is no flink-connector-files dep anymore, this dependency is required to be explicit.

Reply:
I added a separate commit for that, to make it visible